test/test_sql_regexp.py

```diff
@@ -0,0 +1,53 @@
+import pytest
+import anymarkup
+from typing import Any, Dict
+from jsonschema.exceptions import ValidationError
+
+from utils import get_schema_object_validator
+
+valid_sql_queries = [
+    "select a from table_a;",
+    "SELECT A FROM TABLE_A;",
+    "select a from table order by id;",
+    "select a from table order by id;",
```

Comment:
nit: is this the same as the line above? Was it meant to be a multi-column query, or something else along those lines?

Reply:
it was intended to be a query with a comment, I removed the comment to just pass the tests bc the regex is not updated yet.